### PR TITLE
[Documentation][Cookbook] Cloudways deployment guide

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
| --------------- | -----
| Branch?         | 1.0 
| New docs?    | yes
| Related tickets | instead of #8821 
| License         | MIT

We've got this guide thanks to @shahroznawaz, I just did minor tweaks and a rebase :)

### CODE REVIEW

File: docs/cookbook/cloudways.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
+How to deploy Sylius to Cloudways PHP Hosting?
+==============================================
+
+Cloudways is a managed hosting platform for custom PHP apps and PHP frameworks such as Symfony, Laravel, Codeigniter, Yii
```
Comment (pamil) [inline docs/cookbook/cloudways.rst]: Missing comma at the end of line

File: docs/cookbook/cloudways.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+CakePHP etc. You can launch the servers on any of the five providers including DigitalOcean, Vultr, AWS, GCE and KYUP containers.
+
+The deployment process of Sylius on Cloudways is pretty much straight forward and easy.
```
Comment (pamil) [inline docs/cookbook/cloudways.rst]: "straight forward" -> "straightforward" IIRC :)

File: docs/cookbook/cloudways.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+You should `signup at Cloudways <https://platform.cloudways.com/signup>`_ to buy the PHP servers from the above mentioned providers. Simply go to the pricing page and choose your required plan.
+You then need to go through the verification process. Once it done login to platform and launch your first Custom PHP application. You can follow the Gif too.
+
+.. image:: ../_images/cloudways-php-server.gif
+    :align: center
+
+Now let’s start the process of installing Sylius on Cloudways.
+
+2. Install Sylius v1.0.0 via SSH
```
Comment (pamil) [inline docs/cookbook/cloudways.rst]: We can skip the version as you'd like to install the latest one, not excatly `1.0.0`

File: docs/cookbook/cloudways.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+the **Master Credentials** heading and login to the SSH terminal:
+
+.. image:: ../_images/cloudways-ssh.png
+    :align: center
+
+After the login, move to the application folder using the ``cd`` command and run the following command to start installing Sylius:
+
+.. code-block:: bash
+
+    $ composer create-project -s beta sylius/sylius-standard myshop
```
Comment (pamil) [inline docs/cookbook/cloudways.rst]: `-s beta` should be removed

File: docs/cookbook/cloudways.rst [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+Keep the rest of the values to default so that the config file will have the defaults Sylius settings.
+If the need arises, you can obviously change these settings later.
+
+
+3. Install Node Dependencies
+----------------------------
+
+Sylius requires several Node packages, which also needs to be installed and updated before setting up the shop. In addition, I also need to start and setup gulp.
+
+Now move to the myshop folder by using ``cd myshop`` and run the following command ``yarn install``. Once the command finishes, run the next command, `yarn run gulp`.
```
Comment (pamil) [inline docs/cookbook/cloudways.rst]: Double backticks missing for the last command

File: docs/cookbook/cloudways.rst [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+    :align: center
+
+Keep the rest of the values to default so that the config file will have the defaults Sylius settings.
+If the need arises, you can obviously change these settings later.
+
+
+3. Install Node Dependencies
+----------------------------
+
+Sylius requires several Node packages, which also needs to be installed and updated before setting up the shop. In addition, I also need to start and setup gulp.
```
Comment (pamil) [inline docs/cookbook/cloudways.rst]: `gulp` -> `Gulp`

File: docs/cookbook/cloudways.rst [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+Now move to the myshop folder by using ``cd myshop`` and run the following command ``yarn install``. Once the command finishes, run the next command, `yarn run gulp`.
+
+4. Populate the Database and Sylius
+-----------------------------------
+
+Comment out the three lines in ``config_dev.yml`` file.
```
Comment (pamil) [inline docs/cookbook/cloudways.rst]: I'm not sure about this one, IMHO install command should be run with `-e prod` instead.

**Comment by pamil** [comment]
Thanks @shahroznawaz and @CoderMaggie! 🎉

**Comment by shahroznawaz** [comment]
Great work Guys @CoderMaggie @pamil @GSadee. You all are Gem :) 👍

**Pull request merged**